### ISSUE TITLE
Add Mimic Matter ability

### ISSUE THREAD

**Issue by Caaz** [Caaz/rimworld-typhon#183]
# Description
Need to create an Ability def for Mimic Matter. This should be usable by any pawn, and be functionally similar to the current Mimic ability in Version 2.

We'll need to patch any hostility checks to not see hidden pawns, and give it to

# Art Needed
Will need ability icon art, that's all! There's likely flecks that are usable for casting.

# Sound Needed
Probably none, I can't imagine hearing a sound for this a ton would be any good.

**Comment by Polioguy** [comment]
![MimicMatter_mimic](https://user-images.githubusercontent.com/63909681/208435173-f620890f-1f01-450b-8cd6-f7ce1197289f.png)
![MimicMatter_Lamp](https://user-images.githubusercontent.com/63909681/208435182-67baf306-4976-47da-80fb-09358f862067.png)
I'm not super happy with either of these, but one of them should work at least for a placeholder graphic

**Comment by Caaz** [comment]
Yeah this is tricky. I feel like the prey icon for it is like a mug which probably isn't immediately recognizable either. Do you happen to have the background graphic you're using there on its own? Maybe I can attempt throwing something together for it or others.

**Comment by Polioguy** [comment]
![Abilityblank](https://user-images.githubusercontent.com/63909681/208652018-7896625c-f7db-4107-be1d-460a8aecfadb.png)
here you go

**Comment by TheErrorInC0de** [comment]
This is what the original Mimic Matter icons look like:
![Mimic Matter Icons](https://user-images.githubusercontent.com/122708353/212803562-991c3007-1b42-4710-98ff-679a32014a58.PNG)
However I was think of what could be done to make them look more RimWorldy as mugs do not exist, so I thought of what was the prop I most associated with RimWorld and I thought of the good old 1x2 table, but I am not an artist and have no idea how this could be pulled off successfully. So I will just keep collecting reference images.

**Comment by Caaz** [comment]
I considered the table to but the biggest thing I'm having things able to mimic are 1x1 tiles which makes things tricky. My next thought was a chair or a boulder, but boulder makes life a little more difficult due to item copying problems. Chair might work though.

This is for sure gonna be the trickiest one.
